### PR TITLE
Wtl ui

### PR DESCRIPTION
**Pull request by project64**
This now has a build for qt and wtl.

Lots of changes and work done, I am sure it is not perfect and I have missed something, but I have tried to make it as close as possible at the qt version

### CODE REVIEW

**Comment by gonetz** [comment]
I've put this PR into https://github.com/gonetz/GLideN64/tree/project64-wtl-ui branch

wtl is incompatible with VS2013, but it is not a problem.
I built it locally with VS2017, works fine.
The only thing to fix before merge to master is appveyor.yml script.
zilmar-spec builds need to be split to
* zilmar-spec-qt
* zilmar-spec-wtl
* zilmar-spec-qt_x64
* zilmar-spec-wtl_x64

File: projects/msvc/GLideNUI-wtl.vcxproj [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|x64">
+      <Configuration>Release</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+  </ItemGroup>
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>{9E05B70F-A294-44A9-A151-B2CC95AA884E}</ProjectGuid>
+    <RootNamespace>GLideNUIwtl</RootNamespace>
+    <WindowsTargetPlatformVersion>8.1</WindowsTargetPlatformVersion>
```
Comment (Jj0YzL5nvJ) [inline projects/msvc/GLideNUI-wtl.vcxproj:24]: This is the "bone of contention", this little line is only used by the UI and breaks compatibility with previous versions of VS

**Comment by Jj0YzL5nvJ** [comment]
[More info regarding `WindowsTargetPlatformVersion` and its possible solutions.](https://developercommunity.visualstudio.com/content/problem/140294/windowstargetplatformversion-makes-it-impossible-t.html)
If only CLI is used, the best solution is to just erase it.

**Comment by gonetz** [comment]
@Jj0YzL5nvJ most of issues you noticed were automatically fixed when I rebased this PR to master.
Please check  https://github.com/gonetz/GLideN64/tree/project64-wtl-ui branch for remaining issues with project files.

**Comment by oddMLan** [comment]
@Jj0YzL5nvJ do you have a Discord account? We'd like to discuss about the suggested changes in the project files.
https://discord.gg/nZwu3Ye

**Comment by project64** [comment]
> I've put this PR into https://github.com/gonetz/GLideN64/tree/project64-wtl-ui branch
> 
@gonetz there is some bug fixes and changes that you and @oddMLan have suggested that I need to fix. 

Those changes, should I do a pull request to the new branch you created or keep updating this branch and update this pull request?

**Comment by oddMLan** [comment]
@gonetz I see you updated the appveyor file, was about to push an update to the project files and the appveyor file to take the wtl build into account properly, so I'll have to revert your commit

**Comment by gonetz** [comment]
@project64 @oddMLan I merged this PR to master. I also updated appveyor script and solution/project files.
So, please do necessary corrections and make pull requests as usual, that is for master.

**Pull request closed without merging**